jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jakarta-server/src/main/java/org/eclipse/jetty/ee10/websocket/jakarta/server/internal/JakartaWebSocketCreator.java

```diff
@@ -169,7 +167,7 @@ public Map<String, Object> getUserProperties()
         catch (Throwable x)
         {
             LOG.warn("Unable to create websocket: {}", config.getEndpointClass().getName(), x);
```

Comment:
I think this really should be a warning though.
We only get here if the class could not be constructed.

Reply:
I don't think there is a point to make a log warn, as nobody would look at the logs if there is a problem instantiating the endpoint class.
We don't log much at warn level to avoid monitoring consoles to go crazy if that mapping gets hammered by a client.

Reply:
Changed to debug.